### PR TITLE
Add QFN-64-1EP 9x9mm with 3.8x3.8mm EP

### PR DESCRIPTION
**Pull request by joshtyler**
Adding QFN-64-1EP 9x9mm with 3.8x3.8mm EP.

This is as defined by Realtek in https://datasheet.lcsc.com/szlcsc/Realtek-Semicon-RTL8211EG-VB-CG_C69264.pdf (p.69), but referenced to JEDEC MO-220.

I haven't used this footprint generator before, so please let me know if I have done something wrong!

Output looks like this:
image

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml [hunk 1]
```diff
@@ -1,3 +1,54 @@
+QFN-64-1EP_9x9mm_P0.5mm_EP3.8x3.8xmm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://datasheet.lcsc.com/szlcsc/Realtek-Semicon-RTL8211EG-VB-CG_C69264.pdf (page 69)'
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml]: Use the #page=xx style that we now use as it means the browser will directly open the correct page (also the pdf page should be used):

https://datasheet.lcsc.com/szlcsc/Realtek-Semicon-RTL8211EG-VB-CG_C69264.pdf#page=77

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+  #body_size_x_min: 8.9
+  body_size_x: 9.0
+  #body_size_x_max: 9.1
+  #body_size_y_min: 8.9
+  body_size_y: 9.0
+  #body_size_y_max: 9.1
+  lead_width_min: 0.18
+  lead_width_max: 0.30
+  lead_len_min: 0.30
+  lead_len_max: 0.50
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml]: nominal sizes for both width and lengths are given so they must be included as well.
(I use the newest dimension entry style as it is the least amount of work)
```yaml
  lead_width: 0.15 .. 0.25 .. 0.3
  lead_len: 0.3 .. 0.4 .. 0.5
```

**Comment by joshtyler** [comment]
Thanks for the feedback, I've read that document and changed to using the newest dimension entry style, as well as implementing the other changes requested.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [3, 3]
+  #heel_reduction: 0.1 #for relatively large EP pads (increase clearance)
+
+  thermal_vias:
+    count: [3, 3]
+    drill: 0.2
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    # EP_num_paste_pads: [2, 2]
```
Comment (joshtyler) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml]: What is the policy on commented out lines, for example these? Should I leave them in or remove them?

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml [hunk 4]
```diff
@@ -1,3 +1,45 @@
...
+    # paste_avoid_via: False
+
+  pitch: 0.5
+  num_pins_x: 16
+  num_pins_y: 16
+  #chamfer_edge_pins: 0.25
+  #pin_count_grid:
+  #pad_length_addition: 0.5
+  #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
+  #include_suffix_in_3dpath: 'False'
```
Comment (joshtyler) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml]: What is the policy on commented out lines, for example these? Should I leave them in or remove them?

**Comment by poeschlr** [comment]
You need to update the footprint pull request.

The commented out lines are not required but do not hurt either.

**Comment by joshtyler** [comment]
I've updated the footprint PR https://github.com/KiCad/kicad-footprints/pull/1858

**Pull request merged**